src/API/Reports/DataStore.php

```diff
@@ -91,6 +162,7 @@ protected function get_cache_key( $params ) {
 	 * @return mixed
 	 */
 	protected function get_cached_data( $cache_key ) {
+		return false;
```

Comment:
Was this left over from testing?

Reply:
Good catch. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2961/commits/851cf8f17b598687e52af0dc4740a84dfa37f7ba